api/v1alpha1/validatedelete_test.go

```diff
@@ -0,0 +1,34 @@
+package v1alpha1
+
+import (
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+var _ = Describe("ValidateDelete methods", func() {
+	It("Should return nil", func() {
+		release := &Release{
+			TypeMeta:   metav1.TypeMeta{},
+			ObjectMeta: metav1.ObjectMeta{},
+		}
+		Expect(release.ValidateDelete()).To(BeNil())
+	})
+
+	It("Should return nil", func() {
+		releaseplan := &Release{
+			TypeMeta:   metav1.TypeMeta{},
+			ObjectMeta: metav1.ObjectMeta{},
+		}
+		Expect(releaseplan.ValidateDelete()).To(BeNil())
+	})
+
+	It("Should returns nil", func() {
+		releaseplanadmission := &Release{
+			TypeMeta:   metav1.TypeMeta{},
+			ObjectMeta: metav1.ObjectMeta{},
+		}
+		Expect(releaseplanadmission.ValidateDelete()).To(BeNil())
```

Comment:
where do these validate functions come from? I don't see them in the *_types.go files

Reply:
They were defined in the webhook files, so as mentioned elsewhere these three specific tests were moved into the existing webhook_test files, and updated to reflect the struct/crd they go with.